build.gradle

```diff
@@ -21,137 +21,185 @@ plugins {
+        ],
+        gettext               : "com.googlecode.gettext-commons:gettext-commons",
+        javax_servlet         : "javax.servlet:servlet-api",
+        javax_validation      : "javax.validation:validation-api",
+        jmock                 : [
+                "org.jmock:jmock",
+                "org.jmock:jmock-junit4",
+        ],
+        validator             : [
+                "org.hibernate.validator:hibernate-validator",
+                "org.hibernate.validator:hibernate-validator-annotation-processor",
+        ],
+        keycloak              : [
+                "org.keycloak:keycloak-servlet-filter-adapter",
+                "org.keycloak:keycloak-adapter-core",
+                "org.keycloak:keycloak-common",
+                "org.keycloak:keycloak-core",
+                "org.keycloak:keycloak-adapter-spi",
+                "org.keycloak:keycloak-servlet-adapter-spi",
+                "org.apache.httpcomponents:httpcore",
+                "org.bouncycastle:bcpkix-jdk15on",
+                "org.bouncycastle:bcprov-jdk15on",
+                "org.keycloak:keycloak-jaxrs-oauth-client",
+                "org.keycloak:keycloak-authz-client",
+        ],
+        hibernate             : [
+                "org.hibernate:hibernate-core",
+                "org.hibernate:hibernate-c3p0",
+                "org.hibernate.common:hibernate-commons-annotations",
+                "net.bytebuddy:byte-buddy",
+                "org.javassist:javassist",
+                "com.fasterxml:classmate",
+                "com.mchange:c3p0",
+                "com.mchange:mchange-commons-java",
+                "org.dom4j:dom4j",
+                "org.jboss:jandex",
+                "org.jboss.logging:jboss-logging",
+                "javax.activation:javax.activation-api",
+        ],
+        ehcache               : [
+                "org.hibernate:hibernate-jcache",
+                "org.ehcache:ehcache",
+                "javax.cache:cache-api",
+        ],
+        artemis               : [
+                "org.apache.activemq:artemis-server",
+                "org.apache.activemq:artemis-core-client",
+                "org.apache.activemq:artemis-commons",
+                "org.apache.activemq:artemis-selector",
+                "org.apache.activemq:artemis-stomp-protocol",
+                "org.apache.activemq:artemis-journal",
+                "io.netty:netty-all",
+                "commons-beanutils:commons-beanutils",
+                "org.jctools:jctools-core",
+                "org.jgroups:jgroups",
+                "org.apache.geronimo.specs:geronimo-json_1.1_spec",
+                "org.apache.johnzon:johnzon-core",
+        ],
+        amqp                  : [
+                "org.apache.qpid:qpid-common",
+                "org.apache.qpid:qpid-client",
+                "org.apache.mina:mina-core",
+                "org.apache.mina:mina-filter-ssl",
+                "geronimo-spec:geronimo-spec-jms",
```

Comment:
Please remove all except for qpid-client. The rest should be transitive.

Reply:
Removed all except "geronimo-spec:geronimo-spec-jms" as it is required for classes in javax.jms package.